### PR TITLE
Contact Form: Use the predefined $title variable in the anchor tag in the button on admin

### PR DESCRIPTION
**Pull request by jeherve**
Fixes #640

### CODE REVIEW

File: modules/contact-form/admin.php [hunk 1]
```diff
@@ -17,13 +17,13 @@ function grunion_menu_alter() {
...
+	$title = esc_html__( 'Add Contact Form', 'jetpack' );
 	$plugin_url = esc_url( GRUNION_PLUGIN_URL );
 	$site_url = esc_url( admin_url( "/admin-ajax.php?post_id={$iframe_post_id}&action=grunion_form_builder&TB_iframe=true&width=768" ) );
 	?>
 
-	<a id="insert-jetpack-contact-form" class="button thickbox" title="<?php esc_html_e( 'Add Contact Form', 'jetpack' ); ?>" data-editor="content" href="<?php echo $site_url ?>&id=add_form">
-		<span class="jetpack-contact-form-icon"></span> <?php esc_html_e( 'Add Contact Form', 'jetpack' ); ?>
+	<a id="insert-jetpack-contact-form" class="button thickbox" title="<?php esc_html_e( $title ); ?>" data-editor="content" href="<?php echo $site_url ?>&id=add_form">
+		<span class="jetpack-contact-form-icon"></span> <?php esc_html_e( $title ); ?>
 	</a>
```
Comment (samhotchkiss) [inline modules/contact-form/admin.php]: @jeherve -- is there a reason I'm missing that we need to double-escape $title?

I would propose that you leave lines 25 & 26 as is, but change 20 to:
`$title = __( 'Add Contact Form', 'jetpack' );`

Thoughts?

**Comment by samhotchkiss** [comment]
Whoops, should have read that someone already suggested that :)

**Comment by samhotchkiss** [comment]
@jeherve : can you update this PR to 7c740fc ?

**Comment by jeherve** [comment]
@samhotchkiss It's all merged into master now, so we should be all good :)

**Comment by samhotchkiss** [comment]
:+1:

**Pull request merged**